packages/flutter/lib/src/material/tabs.dart

```diff
@@ -440,7 +440,8 @@ class TabBar extends StatefulWidget implements PreferredSizeWidget {
 }
 
 class _TabBarState extends State<TabBar> {
-  final ScrollController _scrollController = new ScrollController();
+  ScrollController _scrollController = new ScrollController();
```

Comment:
Oops, I didn't mean to remove the final qualifier. Thanks for pointing that out.